app/features/Type/Notes.py

```diff
@@ -1,13 +1,16 @@
 class Notes:
     def __init__(self):
-        self.notesWithRepetition = []  # {'note': , 'repetition': , 'time': }
+        self.notesWithRepetition = []  # {'note': , 'repetition': , 'percentage': }
         self.notes = []  # {{'note': , 'currentTime': }
         self.totalTime = 0
+        self.totalNotes = 0
 
     def setTotalTime(self, time):
         self.totalTime = time
 
     def getRepetition(self):
+        for i in range(len(self.notesWithRepetition)):
+            self.notesWithRepetition[i]['percentage'] = round((self.notesWithRepetition[i].get('repetition') / self.totalNotes) * 100, 2)
```

Comment:
There's a reason why you're rounding up, why can't you be as specific as possible?